repository/j.json

```diff
@@ -788,6 +788,17 @@
 			]
 		},
 		{
+			"name": "jsFiddle Syntax Theme",
```

Comment:
A Theme is something different for Sublime Text. It should be called a Color Scheme for Sublime users to know what you are referring to.